### PR TITLE
Changes Portuguese Brazilian language

### PR DESCRIPTION
**Pull request by bitcoinuser**
Changes Portuguese Brazilian language

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -141,35 +141,35 @@
...
+    <string name="demo_setupWalletBtn">Configurar carteira</string>
+    <string name="demo_setupWalletFirst">Você precisa primeiro configurar uma carteira.</string>
+    <string name="send_lightningPayment">Enviar pagamento pela lightning</string>
+    <string name="send_onChainPayment">Enviar pagamento pela rede Bitcoin</string>
     <string name="send_sendPayment">Enviar pagamento</string>
     <string name="send_success">Pagamento\nbem sucedido</string>
     <string name="send_fail">Pagamento\nfalhou</string>
-    <string name="receive_generateRequest">Gerar Pedido</string>
-    <string name="receive_generateRequest_failed">Geração de pedido falhou.</string>
+    <string name="receive_generateRequest">Gerar cobrança</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:164]:
```suggestion
    <string name="receive_generateRequest">Cobrar</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -141,35 +141,35 @@
...
     <string name="send_sendPayment">Enviar pagamento</string>
     <string name="send_success">Pagamento\nbem sucedido</string>
     <string name="send_fail">Pagamento\nfalhou</string>
-    <string name="receive_generateRequest">Gerar Pedido</string>
-    <string name="receive_generateRequest_failed">Geração de pedido falhou.</string>
-    <string name="receive_lightning_request">Pedido via Lightning</string>
-    <string name="receive_on_chain_request">Pedido na Blockchain</string>
+    <string name="receive_generateRequest">Cobrar</string>
+    <string name="receive_generateRequest_failed">Geração de cobrança falhou.</string>
+    <string name="receive_lightning_request">Cobrança via Lightning</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:166]:
```suggestion
    <string name="receive_lightning_request">Cobrança via rede Lightning</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -141,35 +141,35 @@
...
-    <string name="send_lightningPayment">Enviar pagamento em lightning</string>
-    <string name="send_onChainPayment">Enviar pagamento na blockchain</string>
+    <string name="setup_connect_heading">Como você deseja usar a ZAP?</string>
+    <string name="setup_connect_local_description">Criar uma carteira neste aparelho.</string>
+    <string name="setup_connect_local_button">Criar carteira</string>
+    <string name="setup_connect_remote_description">Controle se nó remotamente.</string>
+    <string name="setup_connect_remote_button">Conectar ao nó remoto</string>
+    <string name="demo_setupWalletBtn">Configurar carteira</string>
+    <string name="demo_setupWalletFirst">Você precisa primeiro configurar uma carteira.</string>
+    <string name="send_lightningPayment">Enviar pagamento pela lightning</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:159]:
```suggestion
    <string name="send_lightningPayment">Enviar pagamento pela Lightning</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -141,35 +141,35 @@
...
-    <string name="send_lightningPayment">Enviar pagamento em lightning</string>
-    <string name="send_onChainPayment">Enviar pagamento na blockchain</string>
+    <string name="setup_connect_heading">Como você deseja usar a ZAP?</string>
+    <string name="setup_connect_local_description">Criar uma carteira neste aparelho.</string>
+    <string name="setup_connect_local_button">Criar carteira</string>
+    <string name="setup_connect_remote_description">Controle se nó remotamente.</string>
+    <string name="setup_connect_remote_button">Conectar ao nó remoto</string>
+    <string name="demo_setupWalletBtn">Configurar carteira</string>
+    <string name="demo_setupWalletFirst">Você precisa primeiro configurar uma carteira.</string>
+    <string name="send_lightningPayment">Enviar pagamento pela Lightning</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:159]:
```suggestion
    <string name="send_lightningPayment">Enviar pagamento pela rede Lightning</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -141,35 +141,35 @@
...
-    <string name="setup_connect_local_button">Create Wallet</string>
-    <string name="setup_connect_remote_description">Remote control your own node.</string>
-    <string name="setup_connect_remote_button">Connect to Remote Node</string>
-    <string name="demo_setupWalletBtn">Setup Wallet</string>
-    <string name="demo_setupWalletFirst">You first have to setup a wallet.</string>
-    <string name="send_lightningPayment">Enviar pagamento em lightning</string>
-    <string name="send_onChainPayment">Enviar pagamento na blockchain</string>
+    <string name="setup_connect_heading">Como você deseja usar a ZAP?</string>
+    <string name="setup_connect_local_description">Criar uma carteira neste aparelho.</string>
+    <string name="setup_connect_local_button">Criar carteira</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:154]:
```suggestion
    <string name="setup_connect_local_button">Criar carteira</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -141,35 +141,35 @@
...
+    <string name="setup_connect_remote_description">Controle se nó remotamente.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:155]:
```suggestion
    <string name="setup_connect_remote_description">Controle seu nó remotamente.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -141,35 +141,35 @@
...
-    <string name="receive_generateRequest">Gerar Pedido</string>
-    <string name="receive_generateRequest_failed">Geração de pedido falhou.</string>
-    <string name="receive_lightning_request">Pedido via Lightning</string>
-    <string name="receive_on_chain_request">Pedido na Blockchain</string>
+    <string name="receive_generateRequest">Cobrar</string>
+    <string name="receive_generateRequest_failed">Geração de cobrança falhou.</string>
+    <string name="receive_lightning_request">Cobrança via rede Lightning</string>
+    <string name="receive_on_chain_request">Cobrança via rede Bitcoin</string>
     <string name="receive_noActiveChannels">Para poder receber Bitcoin pela rede Lightning, você precisa primeiro abrir canais.</string>
     <string name="receive_noIncomeBalance">Para poder receber Bitcoin pela Lightning Network, pelo menos um dos seus canais precisa ter fundos no lado da contraparte.\n\nPara conseguir isso, você precisa gastar algo na Lightning Network primeiro.\n\nNa área de gerenciar canais, você pode sempre ver o quanto é capaz de enviar ou receber pela Lightning Network.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:169]:
```suggestion
    <string name="receive_noIncomeBalance">Para poder receber Bitcoin pela rede Lightning, pelo menos um dos seus canais precisa ter fundos no lado da contraparte.\n\nPara conseguir isso, você precisa gastar algo na rede Lightning primeiro.\n\nNa área de gerenciar canais, você pode sempre ver o quanto é capaz de enviar ou receber pela rede Lightning.</string>
```

**Comment by michaelWuensch** [comment]
Thanks @bitcoinuser  for your contribution!
Yust let me know when you are finished with the translation. Then I will merge it :)

**Pull request merged**